### ISSUE TITLE
Error at send mail after upgrade Eventum

### ISSUE THREAD

**Issue by phavel** [eventum/eventum#308]
I have CRON on "/bin/process_mail_queue.php" and after upgrade eventum from 3.1.10 to 3.2.3 (with stop on 3.2.0) stopped working send mail and show error:

```
[2017-10-07 23:25:02] app.ERROR: 5.7.0 Must issue a STARTTLS command
first. e196sm12638279wmf.9 - gsmtp
{"traceFile":"/home/users/domain/domain.cz/sub/ukoly/var/log/zf-mail-59d9462e72a3c.json","exception":"[object]
(Zend\\Mail\\Protocol\\Exception\\RuntimeException(code: 0): 5.7.0
Must issue a STARTTLS command first. e196sm12638279wmf.9 - gsmtp\r\n
at
/home/users/domain/domain.cz/sub/ukoly/vendor/zendframework/zend-mail/src/Protocol/AbstractProtocol.php:327)"}
{"url":"/bin/process_mail_queue.php","ip":"2a01:28:ca:63:ec4:7aff:fe45:c2be","http_method":"GET","server":"ukoly.domain.cz","referrer":null,"unique_id":"WdlGLcnWboMOrId7EkXy9wAAARI","file":"/home/users/domain/domain.cz/sub/ukoly/src/Mail/MailTransport.php","line":66,"class":"Eventum\\Mail\\MailTransport","function":"send","memory_usage":"5.75
MB","memory_peak_usage":"5.75 MB","version":"3.2.3","usr_id":""}
```
I didn't change setting after update. 

SMTP setting:
```
  'smtp' => 
  array (
    'from' => 'xyz@domain.cz',
    'host' => 'smtp.gmail.com',
    'port' => '587',
    'auth' => true,
    'username' => 'xyz@domain.cz',
    'password' => 'xyz',
  ),
```

Do you know, where is problem?

**Comment by glensc** [comment]
likely changes from #237 (Eventum 3.2.0)

**Comment by glensc** [comment]
you can add patch from #311:

you need `'ssl' => 'tls'` in your setup:
https://github.com/eventum/eventum/blob/1558770212d80f7c7fefff1cd824c4e92a6ef2e5/src/Mail/MailTransport.php#L133

**Comment by phavel** [comment]
There is now a different problem when I run script "/bin/process_mail_queue.php"

```
[11-Oct-2017 08:12:33 Europe/Prague] app.WARNING: E_WARNING: file_exists(): open_basedir restriction in effect. File(/etc/mailname) is not within the allowed path(s): (/home/users/domain/:/usr/share/pear/) {"code":2,"message":"file_exists(): open_basedir restriction in effect. File(/etc/mailname) is not within the allowed path(s): (/home/users/domain/:/usr/share/pear/)","file":"/home/users/domain/domain.cz/sub/ukoly/src/Mail/MailTransport.php","line":121} {"url":"/bin/process_mail_queue.php","ip":"111.111.111.111","http_method":"GET","server":"ukoly.domain.cz","referrer":null,"unique_id":"Wd22QrYr9gOqnZlQlLT7xgAAA4c","file":null,"line":null,"class":null,"function":"file_exists","memory_usage":"4.5 MB","memory_peak_usage":"4.5 MB","version":"3.2.3","usr_id":""}
[11-Oct-2017 08:12:33 Europe/Prague] PHP Warning:  file_exists(): open_basedir restriction in effect. File(/etc/mailname) is not within the allowed path(s): (/home/users/domain/:/usr/share/pear/) in /home/users/domain/domain.cz/sub/ukoly/src/Mail/MailTransport.php on line 121
[11-Oct-2017 08:12:33 Europe/Prague] app.ERROR: 5.5.2 Syntax error. b190sm15922927wma.41 - gsmtp  {"traceFile":"/home/users/domain/domain.cz/sub/ukoly/var/log/zf-mail-59ddb651ba152.json","exception":"[object] (Zend\\Mail\\Protocol\\Exception\\RuntimeException(code: 0): 5.5.2 Syntax error. b190sm15922927wma.41 - gsmtp\r\n at /home/users/domain/domain.cz/sub/ukoly/vendor/zendframework/zend-mail/src/Protocol/AbstractProtocol.php:327)"} {"url":"/bin/process_mail_queue.php","ip":"111.111.111.111","http_method":"GET","server":"ukoly.domain.cz","referrer":null,"unique_id":"Wd22QrYr9gOqnZlQlLT7xgAAA4c","file":"/home/users/domain/domain.cz/sub/ukoly/src/Mail/MailTransport.php","line":66,"class":"Eventum\\Mail\\MailTransport","function":"send","memory_usage":"4.5 MB","memory_peak_usage":"4.5 MB","version":"3.2.3","usr_id":""}
```

**Comment by phavel** [comment]
Apparently the wrong format of the email and Gmail returns an [error](https://support.google.com/a/answer/3726730?hl=en) (last item). But I do not know what causes it.

**Comment by glensc** [comment]
providing the `traceFile` might help. attach as `.txt` attachment to issue note, can mail me privately as well.

**Comment by phavel** [comment]
I test other [SMTP server](https://napoveda.seznam.cz/cz/email/imap-pop3-smtp/) (other mail provider), but problem is similar. Now is error `5.1.3 Invalid email address`  (before  `5.5.2 Syntax error`)

In detail: 

```
[2017-10-17 11:19:14] app.ERROR: 5.1.3 Invalid email address.
{"traceFile":"/home/users/domain/domain.cz/sub/ukoly/var/log/zf-mail-59e5cb12ba68e.json","exception":"[object]
(Zend\\Mail\\Protocol\\Exception\\RuntimeException(code: 0): 5.1.3
Invalid email address.\r\n at
/home/users/domain/domain.cz/sub/ukoly/vendor/zendframework/zend-mail/src/Protocol/AbstractProtocol.php:327)"}
{"url":"/bin/process_mail_queue.php","ip":"111.111.111.111","http_method":"GET","server":"ukoly.domain.cz","referrer":null,"unique_id":"WeXLEIYk@o0XqIjNL-ci0QAAAM8","file":"/home/users/domain/domain.cz/sub/ukoly/src/Mail/MailTransport.php","line":66,"class":"Eventum\\Mail\\MailTransport","function":"send","memory_usage":"6.25
MB","memory_peak_usage":"6.25 MB","version":"3.2.3","usr_id":""}
```


SMTP setting:
```
  'smtp' => 
  array (
    'from' => 'xyz@domain.cz',
    'host' => 'smtp.seznam.cz',
    'port' => '465',
    'auth' => true,
    'username' => 'xyz@domain.cz',
    'password' => 'xyz',
  ),

```

**Comment by phavel** [comment]
Super! Problem solved by fix #317 (#313).
